src/Bridge/Symfony/Bundle/Resources/public/swagger-ui/oauth2-redirect.html

```diff
@@ -0,0 +1,67 @@
+<!doctype html>
+<html lang="en-US">
+<body onload="run()">
+</body>
+</html>
+<script>
+    'use strict';
+    function run () {
+        var oauth2 = window.opener.swaggerUIRedirectOauth2;
+        var sentState = oauth2.state;
+        var redirectUrl = oauth2.redirectUrl;
+        var isValid, qp, arr;
+
+        if (/code|token|error/.test(window.location.hash)) {
+            qp = window.location.hash.substring(1);
+        } else {
+            qp = location.search.substring(1);
+        }
+
+        arr = qp.split("&")
+        arr.forEach(function (v,i,_arr) { _arr[i] = '"' + v.replace('=', '":"') + '"';})
+        qp = qp ? JSON.parse('{' + arr.join() + '}',
+            function (key, value) {
```

Comment:
I noticed there are whitespace differences from https://github.com/swagger-api/swagger-ui/blob/7cae0d2bdf079fe2f5514f58920eb9e3c1260d14/dist/oauth2-redirect.html

Can you copy the file as it is? Thanks!